control-plane/msp-operator/src/main.rs

```diff
@@ -479,8 +479,16 @@ impl ResourceContext {
             });
         }
 
-        let mut body = HashMap::new();
-        body.insert("disks", self.spec.disks.clone());
+        let mut labels: HashMap<String, String> = HashMap::new();
+        labels.insert(
+            String::from("openebs.io/created-by"),
```

Comment:
can add these strings as const in `common/src/constants.rs` ?

Reply:
Yeah. I would add them.